src/pages/HomePage/MovieCard/styles.js

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const Description = styled.div`
+	background-color: rgba(53, 63, 76, 0.9);
+	display: flex;
+	cursor: pointer;
+	flex-direction: column;
+	height: 100%;
+	margin-top: calc((100vw - 2rem) * 1.333);
+	padding: 0.5rem;
+	transition: 0.2s ease;
+
+	@media (min-width: 430px) {
```

Comment:
Try to put all this kind of variables in a single place, it would be a `constant` file or even the theme of [Styled-components](https://styled-components.com/docs/api#themeprovider). Then you can reuse these values along of applications and make the breakpoints more solid.